scripts/globals/items/shadow_lord_shirt.lua

```diff
@@ -0,0 +1,17 @@
+-----------------------------------------
+-- ID: 26517
+-- Item: Shadow Lord Shirt
+-- Teleports user to Castle Zvahl Keep
+-----------------------------------------
+
+function onItemCheck(target)
```

Comment:
Does the shirt work without having visited the zone before? If so, should make sure the area has been visited before; see [Kingdom Stable Collar](https://github.com/DarkstarProject/darkstar/blob/master/scripts/globals/items/kingdom_stables_collar.lua) for example.

Reply:
I don't know, but I think that's a safe assumption so I made those changes. Thanks!